go/vt/vttablet/tabletmanager/vreplication/vplayer.go

```diff
@@ -256,7 +256,7 @@ func (vp *vplayer) updateHeartbeat(tm int64) error {
 	if err != nil {
 		return err
 	}
-	if _, err := vp.vr.dbClient.Execute(update); err != nil {
+	if _, err := withDDL.Exec(vp.vr.vre.ctx, update, vp.vr.dbClient.ExecuteFetch, vp.vr.dbClient.ExecuteFetch); err != nil {
```

Comment:
OK, I think that makes sense. 🙂 This was the only thing getting called (thus the seen log messages) before the final SwitchTraffic step on the workflow that had existed prior to the upgrade.